modules/ossm-config-openshift-monitoring-kiali.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/observability/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-kiali_{context}"]
+= Configuring OpenShift Monitoring with Kiali
+:context: ossm-config-openshift-monitoring-kiali
+
+//Need some kind of intro
+//Possible file name may change
+//Possible assembly file may change
+//Assemblies, topic map info needs to be worked out still for 3.0, now that docs has a better idea of what is to be included for 3.0 TP1.
+
+//FIX THE STYLE THINGS ONCE DEV HAS REVIEWED/APPROVED THE CONTENT. Content came from different places, so the first priority is ensuring the steps are in the correct order, and are accurate for 3.0 TP1.
+
+The following steps show how to integrate {KialiProduct} with `user-workload monitoring`.
+
+.Prerequisites
+
+* {SMProductName} is installed.
+
+* `user-workload monitoring` is enabled. See "Enabling user-defined projects".
+
+* {KialiProduct} 1.86 is installed.
+
+.Procedure
+
+. Create a `ClusterRoleBinding` for Kiali:
++
+.Example `ClusterRoleBinding` configuration
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kiali-monitoring-rbac
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-monitoring-view
+subjects:
+- kind: ServiceAccount
+  name: kiali-service-account
+  namespace: istio-system
+----
+
+. Find out the revision name of your Istio instance:
++
+.Example configuration
+[source,terminal]
+----
+$ kubectl get istiorevisions.sailoperator.io
+NAME   READY   STATUS    IN USE   VERSION   AGE
+test   True    Healthy   True     v1.21.0   119m9091
+----
```

Comment:
The sail-operator documentation is currently out of date. This step isn't necessary anymore.